src/tree_schema_common.c

```diff
@@ -2613,3 +2590,39 @@ lys_stmt_flags(enum ly_stmt stmt)
 
     return 0;
 }
+
+void ly_sanity_check_module_filename(const struct ly_ctx *ctx, const char *name, const char *revision, const char *filename)
```

Comment:
Uncrustify does not like the return value on the same line as the function declaration. Also, is it really a sanity check? We have no control over how the file is named so I would say it is a "regular" check.

Reply:
Both done.